### ISSUE TITLE
Fetch components from a file that are not component sets

### ISSUE THREAD

**Issue by stacysun-cb** [figma/code-connect#156]
We're trying to use scripts to generate Code Connect files for illustrations. All the illustrations are components instead of component sets in one file. 

When I run `await client.getComponents('https://www.figma.com/...');`, I'm getting the following error and I'm running in node context. Could you suggest if I do anything wrong or is this a bug? 

```
node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module './client/external'
Require stack:
- /node_modules/@figma/code-connect/dist/connect/index_common.js
- /node_modules/@figma/code-connect/dist/react/index_react.js
- /importIcons.js
```
Thanks!

- Code Connect CLI version 1.1.1
- Operating system OSX

**Comment by tomduncalf-figma** [comment]
Hey @stacysun-cb, this was a bug which slipped through in the 1.1.1 release, sorry about that. We just published v1.1.2 which should fix this, please can you test it out and let us know if this fixes your issue?

**Comment by stacysun-cb** [comment]
@tomduncalf-figma it's working now. Thanks!
